src/modules/platform/constants/locale.js

```diff
@@ -17,5 +17,3 @@ const getLocale = () => {
 
   return locale
 }
-
-export default getLocale
```

Comment:
Hmm... I don't really see that this is a constant...
Also: Why did you rename the file?

Reply:
Yeah probably, I just reverted the changes from @klinzo as there is an other way to inject the locale.

Reply:
I rename this to getLocale.js